apps/web-giddh/src/app/accounting/journal-voucher/voucher/voucher.component.ts

```
@@ -2096,19 +2430,28 @@ export class AccountAsVoucherComponent implements OnInit, OnDestroy, AfterViewIn
+            // Add or update selectedAccountIndex in the form
+            if (currentTransaction.get('selectedAccountIndex')) {
+                currentTransaction.get('selectedAccountIndex').setValue(dropdownIndex);
+            } else {
+                currentTransaction.addControl('selectedAccountIndex', this.formBuilder.control(dropdownIndex));
+            }
+        }
+        
+        this.showLedgerAccountList = false;
+            
         if (event?.value === 'createnewitem') {
             return this.addNewAccount();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)